Pollo/ViewControllers/PollsViewController.swift

```diff
@@ -362,15 +362,14 @@ class PollsViewController: UIViewController {
             case .value(let data):
                 guard let pollsResponse = try? self.jsonDecoder.decode(Response<[GetSortedPollsResponse]>.self, from: data), pollsResponse.success else {
                     DispatchQueue.main.async {
-                        let alertController = self.createAlert(title: self.errorText, message: "Failed to join session with code \(code). Try again!")
+                        let alertController = self.createAlert(title: self.errorText, message: "We couldn't find a group with that code. Please try again.", actionTitle: "Okay")
```

Comment:
Seems like in Pollo some alert controllers save their message as a constant above while others have them in the `message:` parameter. What should we adhere to maintain uniformity?

Reply:
@mathewscullin  I agree - just made the message a constant in `PollsViewController`.